src/convert/mapEvent.js

```diff
@@ -1,29 +1,44 @@
-const Y = require('yjs');
-const { toSlatePath } = require('../utils/convert');
+const Y = require("yjs");
+const { Block } = require("slate");
+const { toSlatePath } = require("../utils/convert");
 
-// type MapAction = { action: 'add' | 'update' | 'delete'; oldValue: any };
-// type SetNodeOperationProperties = Pick<
-//   SetNodeOperation,
-//   'newProperties' | 'properties'
-// >;
+const mapInsertNodeOperations = (event) => {
```

Comment:
probably useful to add a function-level comment here that explains the semantics.

Reply:
Thanks!